docs/api.yml

```diff
@@ -525,11 +537,20 @@ components:
     NoteLink:
       type: object
       properties:
+        type:
+          type: string
+          example: splash
         format:
           type: string
           example: application/pdf
         url:
           type: string
+        packaging:
+          type: string
+          example: "<package format identifier string>"
```

Comment:
Real `PF id` better instead?

Reply:
Yes, I think so.  

Typical values: "https://pubrouter.jisc.ac.uk/FilesAndJATS" or "http://purl.org/net/sword/package/SimpleZip"

NOte that packaging may be absent, where links.access: "public".
![image](https://user-images.githubusercontent.com/13220848/76336168-8668ae80-62ed-11ea-992b-eafd9752ed84.png)